src/mesh_tools.rs

```diff
@@ -56,6 +59,47 @@ fn crawl_faces(
     connected_face_indices
 }
 
+/// Joins two mesh geometries into one
```

Comment:
Nit: Missing a period at the end of the sentence.

Reply:
Yeah, I keep adding the periods when you two request them, but in fact it's not a mistake. I wasn't using them for headlines but I shall.